manim/mobject/mobject.py

```diff
@@ -499,13 +541,13 @@ def remove(self, *mobjects: "Mobject") -> "Mobject":
                 self.submobjects.remove(mobject)
         return self
 
-    def __sub__(self, other):
+    def __sub__(self, other: Mobject) -> Mobject:
         raise NotImplementedError
 
-    def __isub__(self, other):
+    def __isub__(self, other: Mobject) -> Mobject:
```

Comment:
Lying about the return value

Reply:
> This is similar to a pull I already made (but you did better), where I made a dedicated file for this. I think the changes should be combined: #2052

Nice, I had a look and I think our changes don't conflict much (if at all) so hopefully we can land them separately.